independent-publisher-connectors/WebexIntegration/apiDefinition.swagger.json

```diff
@@ -0,0 +1,817 @@
+          "423": {
+            "description": "Locked"
+          },
+          "429": {
+            "description": "Too Many Requests"
+          },
+          "500": {
+            "description": "Internal Server Error. Unknown error occurred"
+          },
+          "502": {
+            "description": "Bad Gateway"
+          },
+          "503": {
+            "description": "Service Unavailable"
+          },
+          "default": {
+            "description": "Operation Failed."
+          }
+        },
+        "summary": "Read Meetings",
+        "description": "This Action would read meetings on webex.",
+        "operationId": "ReadMeetings",
+        "parameters": [
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": false,
+            "type": "string",
+            "default": "application/json"
+          },
+          {
+            "name": "password",
+            "in": "header",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "timezone",
+            "in": "header",
+            "required": false,
+            "type": "string"
+          }
+        ]
+      },
+      "post": {
+        "summary": "Create a meeting",
+        "description": "This Action would create a meeting on webex.",
+        "operationId": "CreateAMeeting",
+        "parameters": [
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "description": "Content-Type"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "title": {
+                  "type": "string",
+                  "description": "title",
+                  "title": ""
+                },
+                "agenda": {
+                  "type": "string",
+                  "description": "agenda"
+                },
+                "password": {
+                  "type": "string",
+                  "description": "password"
+                },
+                "start": {
+                  "type": "string",
+                  "description": "start",
+                  "title": ""
+                },
+                "end": {
+                  "type": "string",
+                  "description": "end",
+                  "title": ""
+                },
+                "timezone": {
+                  "type": "string",
+                  "description": "timezone",
+                  "title": "",
+                  "default": "Europe/London"
+                },
+                "enabledAutoRecordMeeting": {
+                  "type": "boolean",
+                  "description": "enabledAutoRecordMeeting",
+                  "title": "",
+                  "enum": [
+                    "",
```

Comment:
@ocarhui the enum type was defined as boolean and you have an empty string. can you please remove it?